src/engine/ruleset/decoders/windows/windows-event-decoder.yml

```diff
@@ -0,0 +1,42 @@
+name: decoder/windows-event-decoder/0
+
+sources: 
+  - decoder/json/0
+
+metadata:
+  description: Parent decoder for Windows events
+  module: Windows
+  compatibility: >
+    This decoder has been tested on Wazuh version 4.3
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/23
+  title: This Windows decoders decodes basic field from Windows events
+  references:
+    - https://learn.microsoft.com/en-us/windows/win32/events/windows-events
+
+check:
+  - ~json.Event: +ef_exists
+
+parse:
+  logpar:
+    - ~json.Event: <~windows/xml/windows>
+
+normalize:
+ - map:
+   - timestamp: $~windows.Event.System.TimeCreated.@SystemTime
+   - event.code: $~windows.Event.System.EventID.#text
+   - event.message: $~parsed_json.Message
+   - event.kind: event
+   - event.provider: $~windows.Event.System.Provider.@Name
+   - host.name: $~windows.Event.System.Computer.#text
+   - winlog.level: +kvdb_get/win_log_level/$~windows.Event.System.Level.#text
```

Comment:
same comment on this one, maybe just checking and mapping the different cases would result in a better erformant decoder. Avoiding use of KVDB

Reply:
Done